cmd/qosserver/main_test.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var server, client *net.UDPConn
+var err error
+
+var _ = Describe("QoS Server tests", func() {
+	BeforeEach(func() {
```

Comment:
this would create the server before and after each test, right? why can't we use just a single instance defined in BeforeSuite and AfterSuite?